packages/commercetools/theme/pages/CheckoutV2/Shipping.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div>
+    <SfHeading
+      :level="3"
+      title="Shipping"
+      class="sf-heading--left sf-heading--no-underline title"
+    />
+    <ShippingForm
+      :isSaving="isSaving"
+      :handleShippingAddressSubmit="handleShippingAddressSubmit"
+      @stepSubmit="handleStepSubmit"
+    />
+  </div>
+</template>
+
+<script>
+import {
+  SfHeading
+} from '@storefront-ui/vue';
+import { useShipping } from '@vue-storefront/commercetools';
+import { onSSR } from '@vue-storefront/core';
+import { reactive } from '@vue/composition-api';
+import ShippingForm from '@/components/Checkout/ShippingForm';
+
+export default {
+  name: 'Shipping',
+  components: {
+    SfHeading,
+    ShippingForm
+  },
+  setup(_, context) {
+    const {
+      save: saveShipping,
+      load: loadShipping,
+      error: shippingError
+    } = useShipping();
+
+    const isSaving = reactive({
+      details: false,
+      method: false
+    });
+
+    onSSR(async () => {
+      await loadShipping();
+    });
+
+    const handleShippingAddressSubmit = async shippingDetails => {
+      isSaving.details = true;
+      await saveShipping({ shippingDetails });
+      if (shippingError.value.save) {
+        return;
+      }
+      // await loadShippingMethod();
+      isSaving.details = false;
+    };
+
+    // const handleShippingMethodSubmit = async shippingMethod => {
```

Comment:
do you need this comment?

Reply:
I leave it like that because it might be useful in the next PR with ShippingMethods. I am going to remove it there if it is useless.

Reply:
put that on your fridge or sth to remember:>